packages/e2e/.eslintrc.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
           "error",
           {
             toBeFalsy: "Use `toBe(false)` instead.",
-            not: null,
```

Comment:
This lint rule is required

Reply:
Actually, you won't need not checks here and you can have only the last with defined.
`.not` don't allow to check values properly which can be a cause of wrong behaviour or it can be a flaky test.

Reply:
Makes sense.